src/main/java/com/spotinst/sdkjava/model/api/ocean/ecs/ApiClusterLaunchSpecification.java

```diff
@@ -0,0 +1,259 @@
+package com.spotinst.sdkjava.model.api.ocean.ecs;
+
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.spotinst.sdkjava.client.rest.IPartialUpdateEntity;
+import com.spotinst.sdkjava.model.bl.ocean.ecs.*;
+import com.spotinst.sdkjava.model.bl.ocean.kubernetes.ClusterIamInstanceProfileSpec;
```

Comment:
Is it correct to include class from ocean.kubernetes?

Reply:
Wrong import. Updated